meson_options.txt

```diff
@@ -0,0 +1,22 @@
+option('yaml',
+  type : 'feature',
+  value: 'enabled',
+  description : 'Whether to support YAML manifests',
+)
+option('dwarf_header_dir',
+  type : 'string',
+  value : '',
+  description : 'The directory where the dwarf.h header is located, if not in include directories.',
+)
+option('documentation',
+  type : 'boolean',
+  value : false,
+  description : 'Whether to build the documentation'
+)
+option('docbook-docs',
```

Comment:
AIUI, Meson style is to use `words_with_underscores` for options. If in doubt, copy the option name from a GNOME project.

Reply:
Will change that.